includes/class-convertkit-resource-forms.php

```diff
@@ -66,6 +66,24 @@ public function get_non_inline() {
 
 	}
 
+
+	/**
+	 * Returns whether any non-inline forms exist in the options table.
+	 *
+	 * @since   2.2.4
+	 *
+	 * @return  bool
+	 */
+	public function non_inline_exist() {
+
+		if ( ! $this->get_non_inline() ) {
```

Comment:
Can this be `return ( !! $this->get_non_inline() )`? More just wondering for my own sake, haven't explored the truthy/falsey values in PHP

Reply:
@noelherrick It can be `return ( ! ! $this->get_non_inline() );` and would absolutely work. I've always been undecided on whether it affects [readability of code](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#clever-code), though. It would pass WordPress' coding standards, so perhaps is more of a reflection of my style than anything.

Happy to go either way on this, let me know.

Reply:
No need to change it 🚢